src/wallet.js

```diff
@@ -151,14 +153,15 @@ class TokenInput extends React.Component {
               placeholder="Enter amount to transfer"
               onChange={e => this.handleChange(e.target.value)}
             />
-            <FormControl.Feedback />
+            <FormControl.Feedback/>
```

Comment:
Can you please remove all these whitespace changes?    Ie, formatting style changes can be a separate PR, much easier to review the "important" bits of this PR

Reply:
done